preact/index.esm.js

```diff
@@ -0,0 +1,19 @@
+import { options } from 'preact';
+import { wrap } from '../';
+export * from '../';
+
+const map = new WeakMap();
```

Comment:
`tsdx` is limiting me a bit too much when setting up additional entry points. Need to drop it at some point.

For now I manually copied the code for the ESM and CJS version. But that's basically the integration with Preact.